app/scripts/modules/core/src/domain/IManagedEntity.ts

```diff
@@ -24,6 +24,49 @@ export interface IManagedResourceSummary {
   };
 }
 
+export interface IManagedEnvironmentArtifact {
+  name: string;
+  type: string;
+  statuses: string[];
+  versions: {
+    current: string;
+    pending: string[];
+    approved: string[];
+    previous: string[];
+    vetoed: string[];
+  };
+}
+
+export interface IManagedEnviromentSummary {
+  artifacts: IManagedEnvironmentArtifact[];
```

Comment:
Similarly to `IManagedArtifactVersionEnvironmentSummary`, what if we just inlined this?